symphony/lib/toolkit/class.json.php

```diff
@@ -5,13 +5,13 @@
  */
 
 /**
- * The `JSONException` class extends the base `Exception` class. It's only
+ * The `SymJSONException` class extends the base `Exception` class. It's only
  * difference is that it will translate the `$code` to a human readable
  * error.
  *
  * @since Symphony 2.3
  */
-class JSONException extends Exception
+class SymJSONException extends Exception
```

Comment:
I'm curious, would a better fix be to only create this exception if it didn't exist? What sort of error messages does the new native `JsonException` give? Are they are descriptive as ours?

Reply:
That was [sugested](https://github.com/symphonycms/symphony-2/issues/2884#issuecomment-466185413) by @nitriques as well... Honestly I haven't tested anything here — I simply needed to make it run on PHP 7.3. Unfortunatley, the PHP docs don't telll me much about how descriptive the exception will be. But if it is desired, I can change the code acoordingly.

Reply:
The code for the native JSONException is almost identical to our implementation: https://github.com/php/php-src/blob/623911f993f39ebbe75abe2771fc89faf6b15b9b/ext/json/json.c#L215

I would not mind dropping our version. We may loose the translation, but will always support all error codes. I do not mind loosing the translation (it is not meant to be user facing anyways).

I'll fix it with a class exists check.

Reply:
Please go ahead! Thanks!

Reply:
Oh, I misunderstood. Should I add a commit to my PR?

Reply:
Done.